### ISSUE TITLE
unable to navigate to split view menu (3 dots) via keyboard

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#44300]
### Description

Unable to use the keyboard to navigate to the split view menu.

Somewhat related to https://github.com/brave/brave-browser/issues/44290, as even if you could open it the options in the menu don't highlight so you won't know which one you're selecting.

### Steps to reproduce

1. Have some split view tabs open
2. Try using the "tab" key to navigate around
3. While tab does navigate between the two tabs, it never lands on the split view menu (3 dots) 

### Actual result

https://github.com/user-attachments/assets/b3366b63-5db3-4958-ba2e-38f3a06ad393

### Expected result

should be able to open the split view 3 dots menu via the keyboard

<img width="229" alt="Image" src="https://github.com/user-attachments/assets/c9636cb9-4405-4924-885b-66bb39c98e91" />

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

1.76.x, 1.77.x

### Channel information

- [ ] release (stable)
- [x] beta
- [x] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc @rebron

**Comment by insilications** [comment]
Related: https://github.com/brave/brave-browser/issues/42204

**Comment by rebron** [comment]
Closing as stale. We've removed the 3 dots menu from the center in favor of split view button on the toolbar as well as the menu item with the tdl in the bottom right corner.
